### ISSUE TITLE
boost_signals removed in boost 1.69, cmake failing for most projects due to this (Trac #2232)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2224]
Hello,

as of boost 1.69, the signals library has been removed and replaced with an apparently slightly changed signals2 library(https://github.com/boostorg/boost/pull/189). Due to this, our cmake fails at the moment on distributions which already deploy boost 1.69 (i.e. Arch Linux).

As an Arch Linux User, this impacts me heavily but I understand that the do not have many people using that distribution. Either way, better to get this fixed sooner than later before we run into a similar boost problem then before.

Migrated from https://code.icecube.wisc.edu/ticket/2232
```json
{
    "status": "closed", 
    "changetime": "2019-01-18T21:21:26", 
    "description": "Hello,\n\nas of boost 1.69, the signals library has been removed and replaced with an apparently slightly changed signals2 library(https://github.com/boostorg/boost/pull/189). Due to this, our cmake fails at the moment on distributions which already deploy boost 1.69 (i.e. Arch Linux).\n\nAs an Arch Linux User, this impacts me heavily but I understand that the do not have many people using that distribution. Either way, better to get this fixed sooner than later before we run into a similar boost problem then before.\n", 
    "reporter": "flauber", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1547846486882000", 
    "component": "cmake", 
    "summary": "boost_signals removed in boost 1.69, cmake failing for most projects due to this", 
    "priority": "normal", 
    "keywords": "cmake arch linux boost 1.69", 
    "time": "2019-01-17T13:39:53", 
    "milestone": "", 
    "owner": "kjmeagher", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20190117T20:41:42: 
* kjmeagher changed owner from "" to "kjmeagher"
* kjmeagher commented:

> In changeset 3130:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="3130"
> Don't load the `signals` module in boost. We don't use it and it was removed from boost. Fixes #2224
> ```


* kjmeagher changed resolution from "" to "fixed"
* kjmeagher changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190118T21:21:26: nwhitehorn commented:

> In changeset 170869:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="170869"
> Don't load the `signals` module in boost. We don't use it and it was removed from boost. Fixes #2224
> ```
